### ISSUE TITLE
Pretty printing of degrees

### ISSUE THREAD

**Issue by dopplershift** [hgrecco/pint#546]
It'd be nice if degreed units (Celsius, Fahrenheit, angles) would pretty print e.g.

```python
'°F'
```

rather than

```python
'degF'
```

Is this technically feasible? If someone can point me in the general direction, I'd be happy to put in a PR.

**Comment by hgrecco** [comment]
That indeed would be nice. Typically this is done by replacing the first alias (the string after the second equal) for the desired definition in the definition file. But this is not entirely correct. 

The issue is that we do not distinguish now between a short form of a unit (which is stored in `symbol` of a definition object) and the pretty printed form. But I would be happy to move in this direction. 

We would need:
1.- Modify the Definition class to accept long and short form for pretty versions
2.- A spec for unit/prefix definition that can be easily parsed
3.- A good fallback rule when the pretty form is not there
4.- Modify the pretty print algorithm to use them

I  can assist if somebody wants to push this forward.

**Comment by cpascual** [comment]
Hi, I am doing the following workaround in my code in order to support both `°C` and `°C` for celsius, and to display `°C` for celsius. [It seems to work for me](https://github.com/cpascual/taurus/blob/degreeC/lib/taurus/core/test/test_units.py), but I am not so sure about its correctness.

```python
UR.define(u'degreeC = kelvin; offset: 273.15  = °C = ºC = degC = celsius')
```


Any comment/suggestion? Something that may backfire and I should be aware of?

Also, are there any plans in supporting this properly in pint?

**Comment by hgrecco** [comment]
it seems ok to me.

**Comment by jules-ch** [comment]
Closed in #811
